runtime/vm/JVMImage.cpp

```diff
@@ -390,144 +377,151 @@ JVMImage::writeImageToFile(void)
 	return true;
 }
 
+JVMImagePortLibrary * 
+setupJVMImagePortLibrary(J9JavaVM *javaVM)
+{
+	PORT_ACCESS_FROM_JAVAVM(javaVM);
+	JVMImagePortLibrary *jvmImagePortLibrary = (JVMImagePortLibrary*)j9mem_allocate_memory(sizeof(JVMImagePortLibrary), OMRMEM_CATEGORY_PORT_LIBRARY);
+
+	OMRPORT_ACCESS_FROM_J9PORT(javaVM->portLibrary);
+	memset(&(jvmImagePortLibrary->portLibrary), 0, sizeof(OMRPortLibrary));
```

Comment:
This is not needed because we memcpy to it the following line. And it covers the same memory region